database/src/main/kotlin/no/nav/su/se/bakover/database/sak/SakRestansRepo.kt

```diff
@@ -42,11 +45,8 @@ internal class SakRestansRepo(
                      select 
                         sak.sakId, 
                         sak.saksnummer, 
-                        s.id, 
-                        case
-                            when søknadinnhold -> 'forNav' ->> 'type' = 'Papirsøknad' then timestamptz(søknadinnhold -> 'forNav' ->> 'mottaksdatoForSøknad')
-                            when søknadinnhold -> 'forNav' ->> 'type' = 'DigitalSøknad' then s.opprettet
-                        end as opprettet,
+                        s.id,
+                        null::timestamp as opprettet,
```

Comment:
select null? Har det noen hensikt?

Reply:
Ja, må visst det for at den UNION-operasjonen skal funke (da må alle delene av unionen matche).

Reply:
Aha, så vi er ikke i opprettet på søknad i for verken papir/digital.